androidshared/src/test/java/org/odk/collect/androidshared/utils/CompressionUtilsTest.kt

```diff
@@ -0,0 +1,58 @@
+package org.odk.collect.androidshared.utils
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers.equalTo
+import org.junit.Assert.assertTrue
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.util.zip.DataFormatException
+
+@RunWith(AndroidJUnit4::class)
+class CompressionUtilsTest {
+    private val text = (
```

Comment:
The text doesn't need to be as long as this right? I think we can also avoid a shared field here and just define our inputs in each test here.

Reply:
Right, fixed.